algorithms/CPlusPlus/Strings/ReverseTheStringWordwise.cpp

```diff
@@ -0,0 +1,44 @@
+// Description :- Given a string, the task is to reverse the order of the words in the given string. 
+// Example :-
+// Input 1:
+//    A = "the sky is blue"
+// Input 2:
+//    A = "this is ib"
+// Output 1:
+//    "blue is sky the"
+// Output 2:
+//    "ib is this"
+
+
+// Time Complexity = O(N), Space Complexity = O(N)
+string Solution::solve(string s) {
```

Comment:
the `solve` function seem to belong to `Solution` namespace which is not present in the code itself. 
either create a Solution namespace to fit this in or more preferably declare the function in global namespace.

```suggestion
string solve(string s) {
```